understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
                </section>
                <section id="change-of-contrast">
                   
-                  <h3>Change of contrast</h3>
+                  <h3>コントラストの変化</h3>
                   
                   
-                  <p>The second part of the Success Criterion's indicator requirements states that an area
-                     of the indicator:
+                  <p>達成基準のインジケータ要件の 2 番目の部分では、インジケータの領域について次のように述べている:
                   </p>
                   
                   
                   <blockquote>
                      
                      <ul>
                         
-                        <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused
-                           states
+                        <li>フォーカスされている状態とフォーカスされていない状態との間で、同一ピクセルに少なくとも 3:1 のコントラスト比がある。
                         </li>
                         
                      </ul>
                      
                   </blockquote>
                   
                   
-                  <p>This requirement measures the change of contrast between the same pixels in different
-                     states. This is different from the Text Contrast and Non-text Contrast Success Criteria,
-                     which measure the contrast between different adjacent pixels in a single state at
-                     a time.
+                  <p>この要件は、異なる状態における同一ピクセル間のコントラストの変化を測定する。これは、一度に単一状態における隣接する異なるピクセル間のコントラストを測定するテキストのコントラスト及び非テキストのコントラストの達成基準とは異なる。
                   </p>
                   
                   
-                  <p>3:1 is the <strong>minimum</strong> allowable change-of-contrast ratio, but the greater the change of contrast between
-                     states, the easier it is for users to see the focus indicator. Authors are encouraged
-                     to make the change-of-contrast ratio as great as possible.
+                  <p>3:1 は許容される<strong>最低限の</strong>コントラスト比の変化であるが、状態間のコントラストの変化が大きいほど、利用者はフォーカスインジケータに気付きやすくなる。コンテンツ制作者は、コントラスト比の変化をできるだけ大きくすることを勧める。
```

Comment:
「勧める」の主語は「コンテンツ制作者」ではないので、少し手を加えてみました。

```suggestion
                  <p>3:1 は許容される<strong>最低限の</strong>コントラスト比の変化であるが、状態間のコントラストの変化が大きいほど、利用者はフォーカスインジケータに気付きやすくなる。コンテンツ制作者には、コントラスト比の変化をできるだけ大きくすることを勧める。
```